src/module-1/card/style.css

```diff
@@ -0,0 +1,149 @@
+.main {
+    width: 80%;
+    display: flex;
+    padding: 50px;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+.container {
+    /* position: relative; */
+    width: 400px;
+    height: 486px;
+    top: 341px;
```

Comment:
rework this one without top 341 you don`t need this one use for position of items flex possibility